ui/shared/ConfirmationDialog.qml

```diff
@@ -9,7 +9,7 @@ ModalPopup {
     id: confirmationDialog
 
     property Popup parentPopup
-    property string btnType: "primary"
+    property string btnType: "warn"
```

Comment:
I don't think the default value should be `warn`, since this component is used in multiple places in the app and not all of them want the "red button"

Reply:
I changed the default value to `warn` because the `color` of the `StatusButton` of this confirmation modal was `danger`.
I think it is meaning the default type is `warn`.
What do you think about it?

Reply:
If there is any case using `primary` color of the button in `ConfirmationModal` really, you are right.
I want to make sure about it with you. :)

Reply:
Hmm, so I checked the places where we use the dialog and the designs and I think you're right, warn can probably be the default one